components/brave_policy/DEPS

```diff
@@ -0,0 +1,6 @@
+include_rules = [
+  "+components/policy/core/browser",
+  "+components/policy/core/common",
+  "+components/policy/policy_constants.h",
```

Comment:
this looks wrong?

Reply:
not needed because of generalized components/policy